default-templates/new-account/index.html

```diff
@@ -29,12 +33,28 @@
     <div class="col-md-12">
       <p class="lead">
         <span>
-          This is a public homepage of {{#if name}}{{name}}, whose WebID is{{else}}a user with WebID{{/if}}
+          This is a public homepage of <a href="{{webId}}">{{#if name}} {{name}} {{/if}}</a>
+        </span>
+      </p>
+    </div>
+  </div>
+
+{{#if tlsCertWebid}}
+  <div class="row" id="update_profile" style="display:none">
+    <div class="col-md-12">
+      <p class="lead">
+        Regarding WebID-TLS+Delegation protocol, add the following RDF statement to your Delegates WebID-Profile Document or click "Update WebID-Profile Doc" to attempt this action right now.
```

Comment:
Is this fixed in the template?

Reply:
Rollback fix to original version

Reply:
What do you mean ? The file default-templates/new-account/index.html is template for user's index.html